SS330_User_Manual.tex

```diff
@@ -2,7 +2,8 @@
 %Options added to improve pdf accessibility:
 \PassOptionsToPackage{enable-debug,check-declarations}{expl3}
 \RequirePackage{pdfmanagement-testphase}
```

Comment:
I am fairly positive that line 4 needs to be line 1 for the accessibility to work.